Packs/PaloAltoNetworks_IoT/Playbooks/playbook-iot-with-servicenow-playbook.yml

```diff
@@ -0,0 +1,357 @@
+    type: condition
+    task:
+      id: be71d420-9199-41ce-8cee-31f7db39a642
+      version: -1
+      name: ServiceNow Table Existed?
+      description: Check if a given value exists in the context. Will return 'no'
+        for empty empty arrays. To be used mostly with DQ and selectors.
+      scriptName: Exists
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "9"
+      "yes":
+      - "8"
+    scriptarguments:
+      value:
+        simple: ${PaloAltoNetworksIoT.raci.r_snow.table}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 1010
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "8":
+    id: "8"
+    taskid: aca86a67-dc23-4563-833f-1307866b85e1
+    type: regular
+    task:
+      id: aca86a67-dc23-4563-833f-1307866b85e1
+      version: -1
+      name: servicenow-create-record
+      description: Creates a new record in the specified ServiceNow table.
+      script: ServiceNow v2|||servicenow-create-record
+      type: regular
+      iscommand: true
+      brand: ServiceNow v2
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      custom_fields:
+        simple: ${PaloAltoNetworksIoT.raci.r_snow.custom_fields}
+      fields:
+        simple: ${PaloAltoNetworksIoT.raci.r_snow.fields}
+      table_name:
+        simple: ${PaloAltoNetworksIoT.raci.r_snow.table}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 275,
+          "y": 1185
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    fieldMapping:
+    - incidentfield: ServiceNow Table Name
+      output:
+        simple: ${PaloAltoNetworksIoT.raci.r_snow.table}
+    - incidentfield: ServiceNow Record ID
+      output:
+        simple: ${ServiceNow.Record.ID}
+    skipunavailable: false
+    quietmode: 0
+  "9":
+    id: "9"
+    taskid: 849b6d99-d818-4697-8eef-f34dcf7068cc
+    type: regular
+    task:
+      id: 849b6d99-d818-4697-8eef-f34dcf7068cc
+      version: -1
+      name: Manual investigation
+      description: Please check the device and incident details.
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1360
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 1550,
+        "width": 605,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: IoTConfigListName
+  value:
+    simple: IOT_CONFIG
+  required: false
+  description: The list name defined in the XSOAR Lists for the RACI and ServiceNow
```

Comment:
Please provide examples to make the description more descriptive
Also the playbook general description should be have more details about this.

Reply:
I am gonna check in one more sentence to this description:

The list name defined in the XSOAR Lists for the RACI and ServiceNow
calculation. Please refer to the description of the command "iot-security-get-raci"
on this list variable.

--------

I have a more complete description in the iot-security-get-raci command.